docs/apis/table.md

```diff
@@ -775,6 +775,8 @@ column names and function names follow Java identifier syntax. Expressions speci
 
 If working with exact numeric values or large decimals is required, the Table API also supports Java's BigDecimal type. In the Scala Table API decimals can be defined by `BigDecimal("123456")` and in Java by appending a "p" for precise e.g. `123456p`.
```

Comment:
No, there is no standard suffix for BigDecimals and `d` is already the suffix for `double`.